src/locutus/model/terminology.py

```diff
@@ -93,6 +129,9 @@ class ChangeType(StrEnum):
         ApprovalRequested = "Approval Requested"
         Approved = "Approved"
         ApprovalDenied = "Approval Denied"
+        AddPreference = "Add API Ontology Preference"

```

Comment:
I see now why you were adding that editor property. However, I don't think these are necessary.